packages/react-reconciler/src/ReactFiberWorkLoop.old.js

```diff
@@ -1383,7 +1383,7 @@ export function renderDidSuspendDelayIfPossible(): void {
     // pending update.
     // TODO: This should immediately interrupt the current render, instead
     // of waiting until the next time we yield.
-    markRootSuspendedAtTime(workInProgressRoot, renderExpirationTime);
+    markRootSuspendedAtTime(workInProgressRoot, renderExpirationTime, false);
```

Comment:
Does your fix work if this is the only one you change?

Reply:
Yeah it should.

Reply:
Let me verify...

Reply:
I inverted the flag and removed this case. So only the other one (renderDidSuspend) is special now.